### PR TITLE
Lint apis/flags directory

### PR DESCRIPTION
**Pull request by mauriciovieira**
This PR is part of #236 endeavor.

### CODE REVIEW

**Comment by mauriciovieira** [comment]
@Digipalvelutehdas/apika-tiimi please review. And please let me rebase and merge.

**Comment by brylie** [comment]
Please resolve conflicts.

**Comment by brylie** [comment]
Great work here. A couple of minor changes and we can merge. I am open to discuss the feedback, if you have other ideas.

File: apis/flags/client/flag.js [hunk 1]
```diff
@@ -25,7 +33,7 @@ Template.flagApiModal.helpers({
     const instance = Template.instance();
 
     // Check if api exists and return boolean
-    return (instance.apiFlag) ? true : false;
+    return !!(instance.apiFlag);
```
Comment (frenchbread) [inline apis/flags/client/flag.js]: I think this one is quite handy. Have not ever used it, but it looks shorter and cleaner.

- [What is `!!`?](http://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript)

Comment (mauriciovieira) [inline apis/flags/client/flag.js]: @brylie this wasn't me, but `eslint --fix`

File: apis/flags/client/flagApiButton/flagApiButton.js [hunk 2]
```diff
@@ -37,6 +44,6 @@ Template.flagApiButton.helpers({
     const apiFlag = instance.apiFlag.get();
 
     // Check if api flag exists
-    return (apiFlag) ? true : false;
+    return !!(apiFlag);
```
Comment (mauriciovieira) [inline apis/flags/client/flagApiButton/flagApiButton.js]: @brylie this wasn't me, but `eslint --fix`. There should be a rule talking about this and thus `--fix` might handle it. I will take a look.

File: apis/flags/client/flag.js (hunk 1 above)
Comment (mauriciovieira) [inline apis/flags/client/flag.js]: `36:33  error  Unnecessary use of boolean literals in conditional expression  no-unneeded-ternary`

File: apis/flags/client/flagApiButton/flagApiButton.js (hunk 2 above)
Comment (mauriciovieira) [inline apis/flags/client/flagApiButton/flagApiButton.js]: http://eslint.org/docs/rules/no-unneeded-ternary

File: apis/flags/collection/schema.js [hunk 3]
```diff
@@ -1,41 +1,48 @@
...
+    },
   },
   apiBackendId: {
     type: String,
-    autoValue: function () {
-
+    autoValue () {
+      let value;
       // Check if the field is not already set
       if (!this.isSet) {
```
Comment (mauriciovieira) [inline apis/flags/collection/schema.js]: I think it is fine and it is not breaking [no-negated-condition rule]( http://eslint.org/docs/rules/no-negated-condition).

Note that we're not doing `if (!this.isNotSet)` (that would be equivalent to [avoid negative conditionals example](https://github.com/ryanmcdermott/clean-code-javascript#avoid-negative-conditionals)

**Comment by mauriciovieira** [comment]
@brylie I just found a no-negated-conditional when rebasing develop, and fixed. The `!isSet` seems to be fine.

**Pull request merged**